doc/developers/contributing.rst

```diff
@@ -952,6 +952,18 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
+Parameters can be passed to the ``fit`` method. Generally, these parameters
+should be restricted to variables that have the shape ``n_samples``.
```

Comment:
"Shape" is the phrase used in the issue this PR is addressing, and is used elsewhere in the documentation. My understanding is that "shape" is a more accurate description of the data, but I'm definitely open to discussion. I agree that an example would be helpful. I will make the change.